### DISCUSSION TITLE
Any plan for a unified cross platform digital pen API?

### DISCUSSION THREAD

**Discussion by kxxt** [dotnet/maui#2397] [Q&A]
Hi, currently I'm building an uwp which uses windows ink feature. I didn't find inkcanvas in the winui3. Maybe it will be added in the future.

Will Maui provide a cross platform digital pen API? That would be awesome!😄

**Comment by davidbuckleyni** [comment]
I no maui is to get a canvas view but again how that would interact with a stylis but with so many devices using them even comercial scanners u raise a valid point.
